lib/addBuffer.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
Add some tests in `addBuffersSpec` to test this new file.

Reply:
Done.